### PR TITLE
Maintenance updates (Support flutter 3.10)

### PR DESCRIPTION
**Pull request by TheHCJ**
Changes I have made:
- Fixed ReadMe ci status badge (#743)
- Removed the mention of the feed on ReadME
- Set `minSdkVersion` to `19`
- Set `useMaterial3` to `true`
- Added more colours by updating `flex_color_scheme`
- Removed `share` in place of `share_plus`
- Cleaned up dependancy overrides

### CODE REVIEW

File: android/app/build.gradle [hunk 1]
```diff
@@ -54,7 +54,7 @@ android {
 
     defaultConfig {
         applicationId "com.jonjomckay.fritter"
-        minSdkVersion 16
+        minSdkVersion 19
```
Comment (jonjomckay) [inline android/app/build.gradle:57]: Is there a reason this has to be bumped to 19? I do still have pings from users on 16, so I'd like to keep Fritter working for them if possible!

File: lib/group/group_model.dart [hunk 2]
```diff
@@ -41,20 +41,21 @@ class GroupModel extends StreamStore<Object, SubscriptionGroupGet> {
       var group = (await database.query(tableSubscriptionGroup, where: 'id = ?', whereArgs: [id])).first;
 
       if (id == '-1') {
-        var subscriptions = (await database.query(tableSubscription))
```
Comment (jonjomckay) [inline lib/group/group_model.dart:44]: Please can you make these whitespace/formatting changes in a separate PR, and/or exclude them from this one? It makes it very difficult to review the _actual_ changes.

`git add -p` is helpful when staging, which allows you choose which chunks to stage and which to ignore.

File: lib/main.dart [hunk 3]
```diff
@@ -483,7 +482,7 @@ class _FritterAppState extends State<FritterApp> {
           blendOnColors: false,
         ),
         visualDensity: FlexColorScheme.comfortablePlatformDensity,
-        useMaterial3: false,
+        useMaterial3: true,
```
Comment (jonjomckay) [inline lib/main.dart:485]: I don't think I'm ready to turn this back on in Fritter, until Flutter's Material 3 support is fully baked. Last time I enabled this, it made some bizarre layout changes, and there were plenty of missing components.

See c25c02c

**Review by jonjomckay — CHANGES_REQUESTED** [review]

File: lib/group/group_model.dart (hunk 2 above)
Comment (TheHCJ) [reply lib/group/group_model.dart:44]: I couldn't stop this from happening as VSCode auto formats on save

File: lib/main.dart (hunk 3 above)
Comment (TheHCJ) [reply lib/main.dart:485]: Ok, I understand I will revert this change

**Pull request closed without merging**